.github/workflows/release.yaml

```diff
@@ -0,0 +1,84 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            release-version:
+              type: string
+              required: true
+              description: 'Version number'
+              default: '0.1.0'
+            is-latest:
+              description: 'Select if the built image should be tagged as latest'
+              required: true
+              type: boolean
+            quay-organization:
+              description: 'Quay organization used to push the built images to'
+              required: true
+              default: 'project-codeflare'
+            python_version:
+              type: string
+              default: "3.10"
+              required: true
+            poetry_version:
+              type: string
+              default: "1.5.1"
+              required: true
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+        steps:
+            - name: Checkout the repository
+              uses: actions/checkout@v3
+            - name: Change version in pyproject.toml
+              run: sed -i "s/\(^version = \"\)\(.*\)\"/\1${{ github.event.inputs.release-version }}\"/" pyproject.toml
+            - name: Install Python
+              uses: actions/setup-python@v4
+              with:
+                python-version: ${{ github.event.inputs.python_version }}
+            - name: Install Poetry
+              uses: abatilo/actions-poetry@v2
+              with:
+                poetry-version: ${{ github.event.inputs.poetry_version }}
+            - name: Run poetry install
+              run: poetry install --with docs
+            - name: Run poetry build
+              run: poetry build
+            - name: Create new documentation
+              run: poetry run pdoc --html -o docs src/codeflare_sdk && pushd docs && rm -rf cluster job utils && mv codeflare_sdk/* . && rm -rf codeflare_sdk && popd
```

Comment:
Is there a reason this does a `poetry run pdoc` instead of just directly running `pdoc`?

Reply:
It will run pdoc in the virtual environment which Poetry created and there are all installed dependencies. I think if you run just "pdoc" it will reply with command not found.